index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap">
+    <title>Student project 1</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+    <!-- Start header -->
+    <header>
+        <div class="container">
+            <a href="#" id="logo">My Logo</a>
+            <nav id="burger-button">
+                <span></span>
+                <span></span>
+                <span></span>
+            </nav>
+            <nav>
+                <ul>
+                    <li><a href="#">Portfolio</a></li>
+                    <li><a href="#">About</a></li>
+                    <li><a href="#">Contact</a></li>
+                </ul>
+            </nav>
+        </div>
+    </header>
+    <!-- End header -->
+    <!-- Start landing -->
+    </div>
+    <section id="landing-page">
+        <div class="container">
+            <hgroup>
+                <h1>I’m Alexandra Glad to see you!</h1>
+                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </hgroup>
+            <div>
+                <p>LET’S CONNECT</p>
+                <ul>
+                    <li><a href="#"><img src="img/twitter.jpg" alt=""></a></li>
+                    <li><a href="#">in</a></li>
```

Comment:
- [ ] [OPTIONAL] you can also export the LinkedIn icon from the provided figma design for a better design. OR you could download it using a [demo  here
](https://drive.google.com/file/d/1YBbxjohyb3B9O5wUy9i_Nsjh-l9-f-jj/view?usp=sharing)